packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -3,17 +3,154 @@
-    getLogger().info(JSON.stringify(wizardResponse))
+    const liveTailSessionConfig: LiveTailSessionConfiguration = {
+        logGroupName: wizardResponse.regionLogGroupSubmenuResponse.data,
+        logStreamFilter: wizardResponse.logStreamFilter,
+        logEventFilterPattern: wizardResponse.filterPattern,
+        region: wizardResponse.regionLogGroupSubmenuResponse.region,
+    }
+    const session = new LiveTailSession(liveTailSessionConfig)
+    if (registry.has(session.uri)) {
+        await prepareDocument(session)
+        return
+    }
+    registry.set(session.uri, session)
+
+    const document = await prepareDocument(session)
+    registerTabChangeCallback(session, registry, document)
+    const stream = await session.startLiveTailSession()
+
+    await handleSessionStream(stream, document, session)
+}
+
+export function closeSession(sessionUri: vscode.Uri, registry: LiveTailSessionRegistry) {
+    const session = registry.get(sessionUri)
+    if (session === undefined) {
+        throw new ToolkitError(`No LiveTail session found for URI: ${sessionUri.toString()}`)
+    }
+    session.stopLiveTailSession()
+    registry.delete(sessionUri)
+}
+
+export async function clearDocument(textDocument: vscode.TextDocument) {
+    const edit = new vscode.WorkspaceEdit()
+    const startPosition = new vscode.Position(0, 0)
+    const endPosition = new vscode.Position(textDocument.lineCount, 0)
+    edit.delete(textDocument.uri, new vscode.Range(startPosition, endPosition))
+    await vscode.workspace.applyEdit(edit)
+}
+
+async function prepareDocument(session: LiveTailSession): Promise<vscode.TextDocument> {
+    const textDocument = await vscode.workspace.openTextDocument(session.uri)
+    await clearDocument(textDocument)
+    await vscode.window.showTextDocument(textDocument, { preview: false })
+    await vscode.languages.setTextDocumentLanguage(textDocument, 'log')
+    return textDocument
+}
+
+async function handleSessionStream(
+    stream: AsyncIterable<StartLiveTailResponseStream>,
+    document: vscode.TextDocument,
+    session: LiveTailSession
+) {
+    try {
+        for await (const event of stream) {
+            if (event.sessionUpdate !== undefined && event.sessionUpdate.sessionResults !== undefined) {
+                const formattedLogEvents = event.sessionUpdate.sessionResults.map<string>((logEvent) =>
+                    formatLogEvent(logEvent)
+                )
+                if (formattedLogEvents.length !== 0) {
+                    await updateTextDocumentWithNewLogEvents(formattedLogEvents, document, session.maxLines)
+                }
+            }
+        }
+    } catch (err) {
+        throw new ToolkitError('Caught on-stream exception')
+    }
+}
+
+function formatLogEvent(logEvent: LiveTailSessionLogEvent): string {
+    if (!logEvent.timestamp || !logEvent.message) {
+        return ''
+    }
+    const timestamp = new Date(logEvent.timestamp).toLocaleTimeString('en', {
```

Comment:
use one of the existing shared datetime format utils. that ensures that we use consistent formatting everywhere in the extension: https://github.com/aws/aws-toolkit-vscode/blob/6fa3a9e554f4aa1916d01d00ea93497ca7205cc5/packages/core/src/shared/utilities/textUtilities.ts#L284-L303

Reply:
Ack, will address.

Reply:
Circling back to this, when I demoed LiveTail to our CWL product manager, it was requested that we shorten the time we display to just HH:MM:SS. The reason being that these events are coming in live, so the additional context of Date and timezone weren't seen as important. 

In that case, is this a required change to make?